etcdserver/etcdserverpb/rpc.proto

```diff
@@ -574,6 +574,10 @@ message WatchCreateRequest {
   }
   // filters filter the events at server side before it sends back to the watcher.
   repeated FilterType filters = 5;
+
+  // If getOldKV is set, created watcher gets the old KV before the event happens.
+  // If the old KV is already compacted, nothing will be returned.
+  bool getOldKV = 6;
```

Comment:
just `prev_kv`? the `get` bit seems unnecessary-- `WithPrevKV()` seems more natural than `WithGetOldKV()`